BUILDING.md

```diff
@@ -38,22 +38,28 @@ future.
 ## Mac OS X
 
 ### Install Build Tools
+First, you need to have Xcode and Xcode Command Line Tools installed. You can follow [this instruction](https://railsapps.github.io/xcode-command-line-tools.html) to install both.
```

Comment:
did a bit search but could not find official doc, normally people just install xcode from app store.